<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Login_page</class>
 <widget class="QWidget" name="Login_page">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>410</width>
    <height>513</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="btn_gotosignup">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>220</y>
     <width>93</width>
     <height>34</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="text">
    <string>注册</string>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="pte_debuginfo">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>270</y>
     <width>387</width>
     <height>221</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>10</y>
     <width>299</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Welcome to TinyQtClient</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_switchMethod">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>220</y>
     <width>161</width>
     <height>34</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="text">
    <string>使用邮箱登录</string>
   </property>
  </widget>
  <widget class="QStackedWidget" name="sw_loginsubpage">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>60</y>
     <width>371</width>
     <height>141</height>
    </rect>
   </property>
   <widget class="QWidget" name="page"/>
   <widget class="QWidget" name="page_2"/>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
